graphics/src/main/kotlin/ktx/graphics/graphics.kt

```diff
@@ -0,0 +1,93 @@
+package ktx.graphics
+
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer
+import com.badlogic.gdx.math.Vector2
+import com.badlogic.gdx.math.Vector3
+
+/** Draws a cube using {@link ShapeType#Line} or {@link ShapeType#Filled}. The pos specify the bottom, left, front corner
+ * of the rectangle. */
+fun ShapeRenderer.box(pos: Vector3, width: Float, height: Float, depth: Float) {
+  this.box(pos.x, pos.y, pos.z, width, height, depth)
```

Comment:
`this.` is redundant.

Does it make sense to add any defaults for the parameters?